tests/scm/scm-test-arity.scm

```diff
@@ -0,0 +1,40 @@
+
+(use-modules (opencog) )
+(use-modules (opencog test-runner))
+(use-modules (opencog python))
+
+(opencog-test-runner)
+(define t "opencog-arity-test")
+
+(test-begin t)
+
+(python-eval "
+from opencog.atomspace import AtomSpace, TruthValue
+from opencog.atomspace import types
+def foo(atspace):
+    TV = TruthValue(0.2, 0.69)
+    atspace.add_node(types.ConceptNode, 'Apple', TV)
+")
+
+(python-call-with-as "foo" (cog-atomspace))
+(test-assert "atom was created" (not (eq? #f (cog-node 'ConceptNode "Apple"))))
+
+(define strength (cog-mean (cog-node 'ConceptNode "Apple")))
+
+(test-assert "strength value is wrong" (< (- 0.2 strength) 0.00001))
```

Comment:
This will fail to actually catch the error: if something screws up, the default TV on Apple is TRUE_TV, which has strength 1 and so `(< (- 0.2 strength) 0.00001))` will be true, even though the the strength is wrong.  You really do want the "abs" for absolute value, in there.